### DISCUSSION TITLE
Problem with set the ms_token

### DISCUSSION THREAD

**Discussion by agil-daat** [davidteather/TikTok-Api#1055] [Q&A]
How to obtain my own ms_token? thanks in advance for your help.

**Comment by Guigeek973** [comment]
Someone said to me that we need to get it from the cookies when we visit tiktok.com but i can't find a way to read the cookies because tiktok is an app on mobile device... if it was eachable via web i can open developer options but in android i don't know how to do.. maybe it will help you, and me ? xd

**Reply by agil-daat** [reply]
I have found the solution. On the TikTok website you must right click, select Inspect and then go to Aplication. Here you will find the value for ms_token

**Reply by Guigeek973** [reply]
Thanks !
